src/components/PriceLine.tsx

```diff
@@ -50,14 +51,21 @@ const PriceLine = ({
     );
     const opacity = React.useMemo(() => (isOverlapping ? opacityOnOverlap : ''), [isOverlapping, opacityOnOverlap]);
 
+    const isBarrierZero = price === '+0.00';
```

Comment:
```suggestion
    const isBarrierZero = parseInt(price) === 0;
```

Reply:
This can't parse integer since I need the value to be 0.00, so I'll pass float instead